template.yaml

```diff
@@ -8,9 +8,9 @@ Metadata:
     Name: Institutions
     Description: Fetching institution and unit data from the Cristin API
     Author: Unit
-    SemanticVersion: 1.0.0
     SpdxLicenseId: MIT
     LicenseUrl: LICENSE
+    Labels: ['${CODEBUILD_RESOLVED_SOURCE_VERSION}', '${GIT_REPO}']
```

Comment:
Which intrinsic function is this one? 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html

Reply:
Environment variables set via codebuild, used by envsubst to tag the resulting SAM application